Firmware/temperature.cpp

```diff
@@ -2811,7 +2824,9 @@ static bool autotune(int16_t cal_temp)
         if(i) break; // we don't need to refine R
         wait(30000); // settle PID regulation
 
-        printf_P(PSTR("TM: %S R estimation @ %dC\n"), verb, cal_temp);
+        sprintf_P(tm_message, PSTR("TM: %S R %dC"), verb, cal_temp);
```

Comment:
since these 3-line sequences are repeated multiple times in the code, I'd suggest making a common function for them:
```
sprintf_P(tm_message, something, ...);
SERIAL_ECHOLN(tm_message);
lcd_setstatus(tm_message);
```
It may save some code...

Reply:
saved 2 bytes by adding lcd_setstatus_serial